optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/ProblemBenchmarkResult.java

```diff
@@ -79,6 +78,9 @@
     private Long variableCount = null;
     private Long problemScale = null;
 
+    @XStreamOmitField // loaded lazily from singleBenchmarkResults
```

Comment:
Comments start with a capital

If this XStream ommitted, shouldn't we XStream omit other fields potentially too?
- Maybe ProblemBenchmarkResult.problemScale
- Definitely SolverBenchmarkResult.averageAverageCalculateCountPerSecond
Please make a separate jira to capture that potential improvement, but let's ignore it for now to avoid going out of scope.

All these fields get refreshed when reading from xml (such as in an Aggregation) through PlannerBenchmarkResult#accumulateResults(...). I presume this field hasNonDefaultSubSingleCount is refreshed too when accumulateResults is called?